### PR TITLE
Sprint12 05 a46727 operating systems supported to metadata

### PR DESCRIPTION
**Pull request by Neve**
operating systems supported to metadata

I will regenerate metadata.json right before merge.

### CODE REVIEW

**Comment by douglaswth** [comment]
Actually you will want to regenerate the metadata.json files on the `release_12H1` after you merge not before.

**Comment by douglaswth** [comment]
Please don't use a loop for the `supports` specification, we should be explicit and readable in this case:

``` ruby
supports "centos", ">= 5.8"
supports "ubuntu", ">= 10.04"
supports "redhat", ">= 5.8"
```

Also, I don't think that just the minimum requirement is what we want; we haven't tested the 12H1 stuff on CentOS 6 for example. We should have an upper bound of support as well.

**Comment by lopaka** [comment]
Since we are only testing these on ubuntu 10.04 and centos 5.8, we should add these as upper bounds.

File: cookbooks/app/metadata.rb [hunk 1]
```diff
@@ -5,6 +5,18 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.rdoc'))
 version          "12.1.0"
 
+
+
+supports "centos", "< 6.0"
+supports "centos", ">= 5.8"
```
Comment (lopaka) [inline cookbooks/app/metadata.rb]: wondering if this is valid by adding 2 supports line with the same OS.  Opscode example shows only one limit input, but not 2.  emailed mailing list to see if there's a way to both an upper and lower limit.

**Pull request merged**